sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -3454,6 +3457,21 @@ private void mergeCase(CaseDataDto leadCaseData, CaseDataDto otherCaseData, bool
 			surveillanceReportDto.setCaze(leadCase.toReference());
 			surveillanceReportFacade.saveSurveillanceReport(surveillanceReportDto);
 		});
+
+		// 10 Activity as case
+		final EpiData otherEpiData = otherCase.getEpiData();
+		if (otherEpiData != null && YesNoUnknown.YES == otherEpiData.getActivityAsCaseDetailsKnown()) {
+			final EpiData leadEpiData = leadCase.getEpiData();
+			if (leadEpiData != null) {
+				if (YesNoUnknown.YES != leadEpiData.getActivityAsCaseDetailsKnown()) {
+					leadEpiData.setActivityAsCaseDetailsKnown(YesNoUnknown.YES);
+					epiDataService.ensurePersisted(leadEpiData);
+				}
+			} else {
+				leadCase.setEpiData(otherEpiData);
```

Comment:
This is a **very** bad idea because it will overwrite ANY epi data changes of the lead case which is not what we want to do. Instead of the logic implemented here, please make sure that
a) if otherCase has at least one activity as case, set activityAsCaseDetailsKnown of leadCase to YES and save the epi data
b) if not cloning, set the epi data of any activity as case of otherCase to the epi data of leadCase and save the activity as case; if cloning, create a copy of each activity as case of otherCase and assign that copy to the epi data of leadCase; see e.g. how it's done for clinical visits above